tests/BlockChypTest/Client/PaymentTest.cs

```diff
@@ -55,8 +56,14 @@ public async void PaymentTest_Charge()
 
                 var response = await blockchyp.ChargeAsync(request);
 
+                Console.WriteLine($"Response: {JsonConvert.SerializeObject(response)}");
```

Comment:
If you want to capture output, you should follow this pattern: https://xunit.net/docs/capturing-output

Reply:
The dotnet CLI seems to swallow echo debugging using this method. Doesn't matter though, I was just using it to troubleshoot something. I'll just nuke it.